tools/shared.py

```diff
@@ -682,6 +682,7 @@ def make_dynCall(sig, args):
 
   @staticmethod
   def make_invoke(sig, named=True):
+    assert('_setThrew' in settings.EXPORTED_FUNCTIONS)
```

Comment:
```suggestion
    assert '_setThrew' in settings.EXPORTED_FUNCTIONS
```

Reply:
I think this should really be `assert 'setThrew' in settings.WASM_EXPORTS`  (note the lack of _ prefix for this list)

Reply:
Actually it looks like we can't do this here because it prevents the later "longjmp support was disabled (SUPPORT_LONGJMP=0), but it is required by the code (either set SUPPORT_LONGJMP=1, or remove uses of it in the project)" from being shown :(

Reply:
OK, removed.